packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -142,58 +219,83 @@ function VisXZoom({
       scaleXMax={zoomConfiguration.maxZoom}
       scaleYMin={zoomConfiguration.minZoom}
       scaleYMax={zoomConfiguration.maxZoom}
-      passive
       top={top}
       width={width}
+      wheelDelta={wheelDelta}
     >
       {_zoom => {
         zoomRef.current = _zoom
```

Comment:
For future reference, the VisX zoom object is documented as `ProvidedZoom` in the zoom type definitions here:
https://github.com/airbnb/visx/blob/86738178e86fdc0feacf83ac96e68724da0627c2/packages/visx-zoom/src/types.ts#L49-L100

VS Code should read those type annotations automatically.